DesktopEdge/Views/Screens/MainMenu.xaml.cs

```diff
@@ -381,6 +379,12 @@ public void Detach() {
             AttachButton.Visibility = Visibility.Visible;
             Arrow.Visibility = Visibility.Collapsed;
         }
+        public void Retach() {

```

Comment:
Reattachment functionality was moved into the mouse event and is no longer required as a stand alone function.